Formula/ntopng.rb

```diff
@@ -1,9 +1,18 @@
 class Ntopng < Formula
   desc "Next generation version of the original ntop"
   homepage "http://www.ntop.org/products/ntop/"
-  url "https://downloads.sourceforge.net/project/ntop/ntopng/ntopng-2.2.tar.gz"
-  sha256 "4fccfc9e9f333addcd3c957b4520c471117bc2df5655d6eabf328c7385fb255e"
-  revision 1
+
+  stable do
+    url "https://github.com/ntop/ntopng/archive/2.4.tar.gz"
+    sha256 "86f8ed46983f46bcd931304d3d992fc1af572b11e461ab9fb4f0f472429bd5dd"
+
+    resource "nDPI" do
+      # tip of 1.8-stable branch; four commits beyond the 1.8 tag
+      url "https://github.com/ntop/nDPI.git",
+        :tag => "1.8-stable",
```

Comment:
All of that makes sense except that it seems to foreclose the possibility of using a branch plus revision for a stable or devel spec, which isn't unreasonable when upstreams have release branches to which they backport fixes, but don't necessarily create separate new tags. Of course, the equivalent can be mirrored on our end by using a sequences of patches, but I'm not sure there's a good philosophical justification for not supporting using a "release branch" for one of the specs as long as the revision is hard coded and upstream has designated it as the release foo branch corresponding to release tag foo.